hack/check-lint.sh

```diff
@@ -1,23 +1,18 @@
-#!/bin/bash
-
-# Copyright 2019 The Kubernetes Authors.
```

Comment:
I suppose this copyright needs to be added back?

Reply:
done.